subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h

```diff
@@ -168,6 +169,50 @@ static inline void hal_trigger_rateoverride_ppi_config(void)
 	nrf_ccm_subscribe_set(NRF_CCM, NRF_CCM_TASK_RATEOVERRIDE, HAL_TRIGGER_RATEOVERRIDE_PPI);
 }
 
+/******************************************************************************/
+#if defined(CONFIG_BT_CTLR_GPIO_PA_PIN) || defined(CONFIG_BT_CTLR_GPIO_LNA_PIN)
+
+#define HAL_ENABLE_PALNA_PPI 5
+#define HAL_DISABLE_PALNA_PPI HAL_ENABLE_PALNA_PPI
+
+static inline void hal_palna_ppi_setup(void)
+{
+	nrf_gpiote_task_t task;
+
+	nrf_timer_publish_set(EVENT_TIMER, NRF_TIMER_EVENT_COMPARE2,
+			      HAL_ENABLE_PALNA_PPI);
+	nrf_radio_publish_set(NRF_RADIO, NRF_RADIO_EVENT_DISABLED,
+			      HAL_DISABLE_PALNA_PPI);
+
+	task = nrf_gpiote_out_task_get(CONFIG_BT_CTLR_PA_LNA_GPIOTE_CHAN);
+	nrf_gpiote_subscribe_set(NRF_GPIOTE, task, HAL_DISABLE_PALNA_PPI);
+}
+
+#endif /* CONFIG_BT_CTLR_GPIO_PA_PIN || CONFIG_BT_CTLR_GPIO_LNA_PIN */
+
+/******************************************************************************/
+#if defined(CONFIG_BT_CTLR_FEM_NRF21540)
+
+#define HAL_ENABLE_FEM_PPI 4
+#define HAL_DISABLE_FEM_PPI HAL_ENABLE_PALNA_PPI
```

Comment:
It looks like it should be:
```suggestion
#define HAL_ENABLE_FEM_PPI 4
#define HAL_DISABLE_FEM_PPI HAL_ENABLE_FEM_PPI
```
Otherwise, the PDN and CSN pins will be toggled on `NRF_TIMER_EVENT_COMPARE3` and `NRF_RADIO_EVENT_DISABLED` but also on `NRF_TIMER_EVENT_COMPARE2` (what would be different from the PPI version) as this event is published to the same channel.